layers/+emacs/org/packages.el

```diff
@@ -715,3 +716,6 @@ Headline^^            Visit entry^^               Filter^^                    Da
         "mtdc" 'spacemacs/org-trello-pull-card
         "mtub" 'spacemacs/org-trello-push-buffer
         "mtuc" 'spacemacs/org-trello-push-card))))
+
+(defun org/init-ox-epub ()
+  (spacemacs|use-package-add-hook org :post-config (require 'ox-epub)))
```

Comment:
Following the convention of the other org packages, this should be defined in two functions:
```elisp
(defun org/pre-init-ox-epub ()
  (spacemacs|use-package-add-hook org :post-config (require 'ox-epub)))
(defun org/init-ox-epub ())
```

(I'm not sure of the rationale why it's done this way, it seems pretty convoluted compared to `:after org` in the use-package declaration.)

Reply:
They should probably also be moved up before `(defun org/pre-init-ox-twbs ()`
so that the ox functions are groupped together.